From 1d9806fe81f8630308d30edd31e3dc911f817293 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Tue, 8 Mar 2011 19:26:02 -0500
Subject: [PATCH] Network: don't be shy

The moment the network panel actually met a real-life NM 0.9, it would
crash and burn...
---
 panels/network/cc-network-panel.c |    5 ++---
 1 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/panels/network/cc-network-panel.c b/panels/network/cc-network-panel.c
index 0d55eb7..eac0825 100644
--- a/panels/network/cc-network-panel.c
+++ b/panels/network/cc-network-panel.c
@@ -1126,11 +1126,10 @@ panel_check_network_manager_version (CcNetworkPanel *panel)
 
                 /* TRANSLATORS: the user is running a NM that is too old and API compatible */
                 message = _("The running NetworkManager version is not compatible (too old).");
-        }
-
         /* nothing to do */
-        if (message == NULL)
+        } else {
                 goto out;
+        }
 
         /* do modal dialog */
         ret = FALSE;
-- 
1.7.4.1

